src/app/components/Header/Notification.jsx

```diff
@@ -81,6 +103,9 @@ class Notification extends React.Component {
       }
     });
   }
+
+  getNumOfUnSeenNotifs = () => this.state.notifications.filter(item => !item.wasSeen).length;
```

Comment:
moved it to inside the render method and defined it as normal variable and not function